extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -187,6 +189,7 @@ export const Action = {
   TOGGLE_UI: 'toggleUi',
   TOGGLE_VIEWPORT_WARNING: 'toggleViewportWarning',
   ADD_NEW_PAGE_ID: 'addNewPageId',
+  UPDATE_PAGE_SIZE: 'updatePageSize',
```

Comment:
Tiny nit: `SET_PAGE_SIZE` to be consistent with the other actions (setPageIds, setNavigationPath...)